bin/bash+

```diff
@@ -21,8 +21,8 @@ if [ "${BASH_SOURCE[0]}" != "$0" ]; then
   bash+:import "$@"
   return $?
 else
-  if [ $# -eq 1 -a "$1" == --version ]; then
-    echo 'bash+   version 0.0.8'
+  if [ $# -eq 1 ] && [ "$1" == --version ]; then
```

Comment:
Should always replace `[...]` with `[[...]]`. You can drop all the LHS double-quoting after that. ie:
```
if [[ $# -eq 1 ]] && [[ $1 == --version ]]; then
```